CMU-313/fall23-nodebb-qwerty-coders
# Added private label UI in list view and on topic

resolves issue #15 

### Changes Made
* Fixed bug where boolean variables are strings by parsing the boolean fields in `src/database/index.js`
* Created the UI label in `themes/nodebb-theme-persona/less/topic.less`
* Imported the private tag tpl files in `themes/nodebb-theme-persona/templates/partials/topics_list.tpl` to show the UI element on the list view and in `themes/nodebb-theme-persona/templates/topic.tpl` to show in the individual topic view.

### How changes have been tested
* Passes `npm run lint` and `npm run test`
* Manual Testing:
   * Created two new topics: one private and one not private.
   * Checked that the private tag is on the private topic and not on the not private topic.
   * Checked the list view to see that all previous posts have the correct tags.
 
### Video Demo

https://github.com/CMU-313/fall23-nodebb-qwerty-coders/assets/65841983/8d417087-34bd-463b-bf23-1b8937934fc3
